packages/xml-views-validation/src/validators/elements/cardinality-of-aggregation.ts

```diff
@@ -0,0 +1,87 @@
+import { filter, forEach, sortBy, map } from "lodash";
+import { XMLElement } from "@xml-tools/ast";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import { getUI5AggregationByXMLElement } from "@ui5-language-assistant/logic-utils";
+import { CardinalityOfAggregationIssue } from "../../../api";
+
+export function validateCardinality(
```

Comment:
`validateAggregationCardinality` ? 

BTW if you will introduce another validation related to cardinality the name would need to be more specific.